Scaling Service, part text and screenshot

OzBob opened the pull request:
updated Scale out screen shot, sorry not all of them ...
and part of the description.

azurecla commented:
Hi __@hungmai-msft__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (hungmai). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

stevestein commented:
@rmca14 @hungmai-msft 
Ross,
Hung is the dev on this feature so this change is ok to merge.

Thanks,
Steve

OzBob committed:
Scaling screenshot

updated Scale out screen shot, sorry not all of them
and part of the description.
